backend/database/models/suspect.py

```diff
@@ -0,0 +1,22 @@
+from backend.database.models._assoc_tables import suspect_officer
+from backend.database.models.officer import Rank
+from .. import db
+
+
+class Suspect(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
+    first_name = db.Column(db.Text)
+    last_name = db.Column(db.Text)
+    race = db.Column(db.Text)
+    gender = db.Column(db.Text)
+    badge = db.Column(db.Text)
+    unit = db.Column(db.Text)  # type?
```

Comment:
Is this something where an enum would make sense or is there too much variation between departments?

Reply:
Yeah, too much variation in the badge and unit values. Most of the time, it should be a number. I'm not certain it always will be though.